Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/ArmoniKPayloadSerialization.cs

```diff
@@ -35,7 +37,7 @@ public static void ShouldSerialize()
     var payload = new ArmonikPayload
                   {
                     MethodName          = "Test",
-                    ClientPayload       = "Payload"u8.ToArray(),
+                    ClientPayload       = Encoding.ASCII.GetBytes("Payload"),
```

Comment:
why is this changing ?

Reply:
that's because in the root build.props, langage is latest so 11 today. You need to install dotnet 7

Reply:
Since we expect to build in dotnet 6 (as it's the latest target in the csproj), shouldn't we keep it as dotnet 6 compatible ?

Reply:
I think so, we should fix the langage version to 10 but for the whole project. Thus, not this pr